app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Contestants</h1>
+
+<% @contestants.each do |contestant| %>
+  <div id="cont-<%= contestant.id %>">
```

Comment:
Great work wrapping this in a div so that you can test where text is appearing more explicitly!